### PR TITLE
Structure Generator to run

### PR DESCRIPTION
**Pull request by mariari**

### CODE REVIEW

File: library/Sexp/src/Juvix/Sexp/Structure/generator.lisp [hunk 1]
```diff
@@ -370,7 +368,7 @@ and a rhs that may contain a guard, so no = is assumed for the rhs"
 
   (generate-haskell "LetHandler" (repeat 3 "sexp") ":let-handler")
 
-  (generate-haskell "Handler" '("sexp" "letRet" "letOp") ":let-handler" :list-star t))
+  (generate-haskell "LetHandler" '("sexp" "letRet" "letOp") ":let-handler" :list-star t))
```
Comment (cyberglot) [inline library/Sexp/src/Juvix/Sexp/Structure/generator.lisp:367]: This one should be `Handler`.

File: library/Sexp/src/Juvix/Sexp/Structure/Frontend.hs [hunk 2]
```diff
@@ -1158,127 +1174,13 @@ toVia :: Sexp.T -> Maybe Via
...
-
-instance Structure Do where
-  to = toDo
-  from = fromDo
-
-instance Structure Via where
-  to = toVia
-  from = fromVia
+fromVia (Via sexp1 sexp2) =
+  Sexp.list [Sexp.atom nameVia, sexp1, sexp2]
```
Comment (cyberglot) [inline library/Sexp/src/Juvix/Sexp/Structure/Frontend.hs:1207]: It would be better to just generate code that instantiate each structure than doing this.

**Comment by mariari** [comment]
Currently the code is not ready yet because:

- The `defhandler` parser is different from what it was before

**Comment by mariari** [comment]
This PR removes the `:ops` part of `:def-op` for now, and makes it an implicit list. We can change this back later, but this was what was easiest to get compiling, see the differences in the tests

**Comment by mariari** [comment]
It is probably best to review this commit by commit

File: library/Translate/src/Juvix/Translate/Pipeline/TopLevel.hs [hunk 3]
```diff
@@ -370,9 +370,9 @@ transDo (Types.Do'' bs) =
 
 transDoBody :: Types.DoBody -> Sexp.T
 transDoBody (Types.DoBody Nothing expr) =
-  Sexp.list [Sexp.atom ":do-body", transComp expr]
+  transComp expr
 transDoBody (Types.DoBody (Just n) expr) =
-  Sexp.list [Sexp.atom ":do-body-binder", Sexp.atom (NameSymbol.fromSymbol n), transComp expr]
+  Sexp.list [Sexp.atom ":<-", Sexp.atom (NameSymbol.fromSymbol n), transComp expr]
```
Comment (cyberglot) [inline library/Translate/src/Juvix/Translate/Pipeline/TopLevel.hs:375]: 🤩

**Comment by cyberglot** [comment]
Looks good to me, good job 👍

**Review by cyberglot — APPROVED** [review]

**Pull request merged**